cmd/backup/backup.go

```diff
@@ -38,8 +38,18 @@ var BackupCmd = &cobra.Command{
 
 var getBackupCmd = &cobra.Command{
 	Use:   "get",
-	Short: "Get list of existing backups available for a cluster in YugabyteDB Managed",
-	Long:  "Get list of existing backups available for a cluster in YugabyteDB Managed",
+	Short: "Get backup details in YugabyteDB Managed",
```

Comment:
The cli is still quite new, should just remove the command fully ?

Reply:
Why should we remove the command when it is working?

Reply:
We will properly run through a deprecation cycle. We will remove it with the next release.